example/testapp/models.py

```diff
@@ -10,9 +10,15 @@ def __unicode__(self):
         return self.plate
 
 
+class m2mprint:
```

Comment:
apply the same suggestions I gave you in https://github.com/gregmuellegger/django-sortedm2m/pull/101/files#r108227092